youtube_dl/extractor/dailymotion.py

```diff
@@ -171,8 +171,7 @@ def _real_extract(self, url):
              r'__PLAYER_CONFIG__\s*=\s*({.+?});'],
             webpage, 'player v5', default=None)
         if player_v5:
-            player = self._parse_json(player_v5, video_id)
-            metadata = player['metadata']
+            metadata = self._download_json('http://www.dailymotion.com/player/metadata/video/%s?integration=inline&GK_PV5_NEON=1' % video_id, video_id)
```

Comment:
1. This must only take place when it's not available from player JSON.
2. Query must be passed as `query`.